odl/operator/default_ops.py

```diff
@@ -44,10 +44,9 @@ def __init__(self, space, scalar):
         Parameters
         ----------
         space : `LinearSpace`
-            Space of elements which the operator is acting on
-        scalar : `LinearSpace.field` `element`
-            An element of the field of the space which vectors are
-            scaled with
+            Space of elements on which this operator acts.
+        scalar : `LinearSpace.field` `element-like`
```

Comment:
Good catch, we should have an eye on all these unspecific types and corresponding checks - many can be replaced by `vec in space` checks.